### DISCUSSION TITLE
Is there any way to change image of card onHover?

### DISCUSSION THREAD

**Discussion by RitikDua** [pmndrs/react-spring#1121] [Support]
I implemented it using props. onHover it receives new props image to set as backgroundImage but it is not working as elegant as needed. It takes a minute time delay during it first execution. Is there any specific to do so.
I am willing to put a transition effect of fade during change of image so the time gap of image could be hindered.

**Comment by lilpolymath** [comment]
Hi @RitikDua can you put your current code on a service like Codesandbox or post the snippet? That would help with solving the issue.
